containers/daap-python-base/tests/unit/versioning_test.py

```diff
@@ -434,63 +434,36 @@ def test_success(
         schema_list = ["schema0"]
         with patch("glue_and_athena_utils.glue_client", glue_client):
             version_manager.update_metadata_remove_schemas(schema_list=schema_list)
-            schema_prefix = f"{data_product_name}/v2.0/metadata.json"
+            schema_prefix = f"{data_product_name}/v3.0/metadata.json"
             response = s3_client.list_objects_v2(
                 Bucket=self.bucket_name,
                 Prefix=schema_prefix,
             )
         assert response.get("KeyCount") == 1
 
-    def test_invalid_schemas(self, data_product_name):
-        version_manager = VersionManager(data_product_name, logging.getLogger())
-        schema_list = ["schema3", "schema4"]
-
-        with pytest.raises(InvalidUpdate) as exc:
-            version_manager.update_metadata_remove_schemas(schema_list=schema_list)
-        assert (
-            str(exc.value)
-            == "Invalid schemas found in schema_list: ['schema3', 'schema4']"
-        )
-
-    def test_glue_table_not_found(
```

Comment:
The code used to be strict on validating if a schema existed in metadata schemas and would return, however it was pointed out that the could be scenarios where data files or tables were present, but a schema was not so this was not an optimal approach. The code will log schemas that are not present in the metadata but will attempt to remove tables and files for any provided schema value. Hence there is no longer a need to validate the supplied schema.

Reply:
I think I get it now - there was no way to delete an 'orphaned' table for a Data Product through the API.

I think it may be preferable to at least return the failure message to the user `'{schema_name} schema metadata not found, {schema_name} schema metadata not deleted'`, ideally together with a message of the glue table being deleted to give the full context of what's happened (as part of the response message):

```python
# schema not found
f"{schema} schema metadata not found, {schema} schema metadata not deleted"
f"{data_product}.{schema} table was successfully deleted, and can no longer be queried"

# schema found
f"{schema} schema metadata found, {schema} schema metadata successfully deleted"
f"{data_product}.{schema} table was successfully deleted, and can no longer be queried"
```